test/e2e/tests/account/incremental-security.spec.js

```diff
@@ -1,35 +1,24 @@
 const { strict: assert } = require('assert');
-const { convertToHexValue, withFixtures, openDapp } = require('../../helpers');
+const { withFixtures, openDapp } = require('../../helpers');
 const FixtureBuilder = require('../../fixture-builder');
 
 const WALLET_PASSWORD = 'correct horse battery staple';
 
 describe('Incremental Security', function () {
-  const ganacheOptions = {
-    accounts: [
-      {
-        secretKey:
-          '0x250F458997A364988956409A164BA4E16F0F99F916ACDD73ADCD3A1DE30CF8D1',
-        balance: convertToHexValue(0),
-      },
-      {
-        secretKey:
-          '0x53CB0AB5226EEBF4D872113D98332C1555DC304443BEE1CF759D15798D3C55A9',
-        balance: convertToHexValue(25000000000000000000),
-      },
-    ],
-  };
-
```

Comment:
anvil doesn't handle priv keys like this, we remove this and we seed the specific account below